ragen/env/search/env.py

```
@@ -0,0 +1,550 @@
+                            "target": example['golden_answers']
+                        },
+                        "data_source": self.config.data_source
+                    })
+                return data
+                
+            elif self.config.data_source == 'strategyqa':
+                # StrategyQA has special handling in Search-R1
+                # Note: StrategyQA may need custom path handling - adjust as needed
+                dataset = datasets.load_dataset('json', data_files="/path/to/strategyqa/test_correct.jsonl")
+                # For StrategyQA, use the dataset as-is (typically test split)
+                data_split = list(dataset.values())[0]
```

Ruff (0.14.10)

Prefer `next(iter(dataset.values()))` over single element slice

Replace with `next(iter(dataset.values()))`

(RUF015)